safe-global/safe-wallet-web
# fix: Set pending state for batched transactions

## What it solves

Resolves [RC issue](https://github.com/safe-global/safe-wallet-web/pull/4290#issuecomment-2387667536)

Fixes a regression introduced in #4158 

## How this PR fixes it

- Sets `nonce` when emitting tx events from the batch execution

## How to test it

1. Queue and sign at least 2 transactions
2. Bulk execute them
3. Observe the Execute button changing to a processing state

## Checklist
